x-pack/plugins/maps/public/components/layer_panel/layer_settings/layer_settings.js

```diff
@@ -146,17 +117,14 @@ export function SettingsPanel(props) {
 
   return (
     <Fragment>
-
-      {renderLayerErrors()}
-
       <EuiPanel>
         <EuiFlexGroup>
           <EuiFlexItem>
             <EuiTitle size="xs">
               <h5>
                 <FormattedMessage
-                  id="xpack.maps.layerPanel.settingsPanel.settingsTitle"
-                  defaultMessage="Settings"
+                  id="xpack.maps.layerPanel.layerSettingsTitle"
```

Comment:
@Bamieh how does this work, going forward wrt localization? Will the framework auto-detect all deltas as we rename and introduce new message-ids, and new translations be generated?